HW_3/templates/form.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+
+{% block title %}
+    {{ title }}
+{% endblock %}
+
+{% block content %}
+
+<form method="POST" action="{{ url_for('registration') }}">
+    {{ form.csrf_token }}
+    {% for field in form if field.name != 'csrf_token' %}
+    <p>
+        {{ field.label }}<br>
+        {{ field }}
+        {% if field.errors %}
+        <ul>
+            {% for error in field.errors %}
+            <li>{{ error }}</li>
+            {% endfor %}
+        </ul>
+        {% endif %}
+    </p>
+    {% endfor %}
+    <p>
+      <input type="submit" value="registration">
+    </p>
```

Comment:
То эту кнопку можно не создавать, так как она будет в объекте формы.
```suggestion
    <p>
        {{ form.submit() }}
    </p>
```